app/views/shared/_breadcrumbs.html.erb

```diff
@@ -1,5 +1,3 @@
-<% # THIS IS PART OF EDUCATION NAVIGATION A/B TESTING - PLEASE KEEP IN SYNC WITH +new_navigation.erb %>
```

Comment:
Do we still need a warning about keeping templates in sync?
Seems like this logic could be moved into a presenter? Maybe in a future PR.

Reply:
I think people who'll modify those templates will be aware of the different variants, so I don't think we need the warning.